### PR TITLE
SNOW-1582 : In Snowshare load data is not available. We need to popul…

### PR DESCRIPTION
**Pull request by govindunravel**
…ate load data in snowshare mode.

- realtime query based on warehouse name
[SNOW-1582](https://unraveldata.atlassian.net/browse/SNOW-1582)

In Snowshare load data is not available. We need to populate load data in snowshare mode.

[SNOW-1582]: https://unraveldata.atlassian.net/browse/SNOW-1582?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: script/replication/replication_unravel.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+--Step-1 (this is not for customer) Receive the data in recipient account (Sql mode).
+--Validate that the inbound share is available to the consumer account.
+Show shares like '%UNRAVEL%';
```
Comment (renukaprasadunravel) [inline script/replication/replication_unravel.sql:4]: Why this statement required here?

File: script/replication/replication_procedure_task.sql [hunk 2]
```diff
@@ -30,14 +36,6 @@ CREATE OR REPLACE TASK replicate_history_query
 AS
 CALL REPLICATE_HISTORY_QUERY('UNRAVEL_SHARE','SCHEMA_4823_T',2);
 
-
--- create warehouse replicate Task
```
Comment (renukaprasadunravel) [inline script/replication/replication_procedure_task.sql:34]: Can you keep this task as well? If something goes wrong with RT_Query task, then warehouse data not available for Query_History also. @viswa-unravel pls help to confirm this behavior.

**Review by renukaprasadunravel — APPROVED** [review]
LGTM

File: script/replication/replication_unravel.sql (hunk 1 above)
Comment (govindunravel) [reply script/replication/replication_unravel.sql:4]: It is for filtering all the share records which has UNRAVEL as a part of it. It is just for viewing the shares.

File: script/replication/replication_procedure_task.sql [hunk 3]
```diff
@@ -1,20 +1,26 @@
...
 => '1', days => '2');
 
---One time execution for health check for 7 days
-CALL CREATE_TABLES('UNRAVEL_SHARE','SCHEMA_4823_T');
-CALL REPLICATE_ACCOUNT_USAGE('UNRAVEL_SHARE','SCHEMA_4823_T',7);
-CALL WAREHOUSE_PROC('UNRAVEL_SHARE','SCHEMA_4823_T');
-CALL REPLICATE_REALTIME_QUERY('UNRAVEL_SHARE','SCHEMA_4823_T',10);
-CALL CREATE_QUERY_PROFILE(dbname => 'UNRAVEL_SHARE', schemaname => 'SCHEMA_4823_T', credit
-=> '1', days => '7')
+--Select one procedure from below two procedure
```
Comment (viswa-unravel) [inline script/replication/replication_procedure_task.sql:10]: Select one procedure from REPLICATE_REALTIME_QUERY or REPLICATE_REALTIME_QUERY_BY_WAREHOUSE based on requirement

File: script/replication/replication_procedure_task.sql [hunk 4]
```diff
@@ -1,20 +1,26 @@
...
+
+--Select and run this procedure if you wish to get real-time queries for all warehouses.
```
Comment (viswa-unravel) [inline script/replication/replication_procedure_task.sql:12]: /* use this comment

File: script/replication/replication_procedure_task.sql [hunk 5]
```diff
@@ -1,20 +1,26 @@
...
+--It will select a maximum of 10,000 real-time queries across all warehouses at intervals of 48 hours.
+
+--CALL REPLICATE_REALTIME_QUERY('UNRAVEL_SHARE','SCHEMA_4823_T', 48);
```
Comment (viswa-unravel) [inline script/replication/replication_procedure_task.sql:15]: Enable by default

**Review by viswa-unravel — COMMENTED** [review]
pls check the comments

**Review by viswa-unravel — APPROVED** [review]
LGTM

**Pull request merged**